coral-schema/src/test/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverterTests.java

```diff
@@ -423,8 +423,8 @@ public void testLateralViewArrayWithComplexType() {
 
   @Test
   public void testLateralViewMap() {
-    String viewSql = "CREATE VIEW v AS " + "SELECT bl.Id AS Id_View_Col, t.Col1, t.Col2 " + "FROM baselateralview bl "
-        + "LATERAL VIEW explode(bl.Map_Col_String) t as Col1, Col2";
+    String viewSql = "CREATE VIEW v AS " + "SELECT bl.Id AS Id_View_Col, t.col1, t.col2 " + "FROM baselateralview bl "
+        + "LATERAL VIEW explode(bl.Map_Col_String) t as col1, col2";
```

Comment:
The existing test case has a bug (I think) that in the Avro schema, the column names are `Id_View_Col`, `col1`, `col2`: the first one the same as the query, the latter two somehow lower-cased.  The existing code somehow would produce this inconsistent cases for different parameters, until my patch when the code generates Avro schema with exactly the cases as in the query.

So I have to either change the query or change the Avro schema to make the test pass.  I decide to change the query since that's easier.